README.md

```diff
@@ -182,17 +177,17 @@ This is an application that connects to IoT Hub using the service SDK offering t
 
 ### 1. Review the existing samples and docs
```

Comment:
Feedback for `Thermostat Device` section (which isn`t in this PR).  I think someone new-ish to PNP needs a bit more motivation, especially the term `model`.  Something along lines of (needs wordsmithing) "The device represents itself as a PnP Thermostat model.  This model implements multiple interfaces, one of which is the thermostat."  I get not wanting to re-document PnP here, but I think confusion risk on this esp given similar naming merits something.

Reply:
I want to make a clear distinction between the _device_ (Thermostat) and the _component_ (TempSensor), but I get your point. Updating, hope the samples are better than words here.